### DISCUSSION TITLE
z_ao_eu cannot load error

### DISCUSSION THREAD

**Discussion by tr3kita** [kubilus1/autoortho#458] [General]
good evening,
i keep getting this error
log :
[Log.txt](https://github.com/kubilus1/autoortho/files/13403725/Log.txt)
i have dokan and winfsp AO 0.7.1

**Comment by Jeff2122** [comment]
You have AO disabled according to your log. 

You also have many plugins installed, 13 including the last entry from an E-Jet just before the apparent crash. 

If there is even a remote possibility for you to install a second copy of X-Blame 12 somewhere on your system, you should do it and ONLY install AO and get it working properly. No plugins, no other scenery but AO. Then port it to your good copy.

**Reply by hawajka76** [reply]
Hello, I've been getting the below perhaps related error type after about 2-3 hours of running autoortho. Is this setting related by any chance? This is on Ubuntu with XP11:


2:57:49.035 I/SCN: DSF load time: 236232 for file Custom Scenery/Global Airports/Earth nav data/+40+010/+46+013.dsf (0 tris, 0 skipped for 0.0 m^2)
2:57:49.272 I/SCN: DSF load time: 217703 for file Custom Scenery/yAutoOrtho_Overlays/Earth nav data/+40+010/+46+013.dsf (0 tris, 0 skipped for 0.0 m^2)
2:57:49.292 I/SCN: DSF load time: 510728 for file Custom Scenery/Global Airports/Earth nav data/+40+010/+46+011.dsf (0 tris, 0 skipped for 0.0 m^2)
2:57:49.768 I/SCN: DSF load time: 437274 for file Custom Scenery/yAutoOrtho_Overlays/Earth nav data/+40+010/+46+011.dsf (0 tris, 0 skipped for 0.0 m^2)
2:58:10.525 I/SCN: DSF load time: 21225328 for file Custom Scenery/z_ao_eur/Earth nav data/+40+010/+46+013.dsf (2231968 tris, 73 skipped for -44.8 m^2)
2:58:10.590 I/SCN: DSF load time: 20659218 for file Custom Scenery/z_ao_eur/Earth nav data/+40+010/+46+011.dsf (2236839 tris, 58 skipped for -261.4 m^2)
2:58:12.487 I/SCN: DSF load time: 148137 for file Custom Scenery/Global Airports/Earth nav data/+40+010/+46+012.dsf (0 tris, 0 skipped for 0.0 m^2)
2:58:12.597 I/SCN: DSF load time: 551990 for file Custom Scenery/Global Airports/Earth nav data/+40+010/+46+014.dsf (0 tris, 0 skipped for 0.0 m^2)
2:58:12.707 I/SCN: DSF load time: 181840 for file Custom Scenery/yAutoOrtho_Overlays/Earth nav data/+40+010/+46+012.dsf (0 tris, 0 skipped for 0.0 m^2)
2:58:12.885 I/SCN: DSF load time: 278806 for file Custom Scenery/yAutoOrtho_Overlays/Earth nav data/+40+010/+46+014.dsf (0 tris, 0 skipped for 0.0 m^2)
2:58:26.548 E/SCN: Failed to find resource 'terrain18.ter', referenced from scenery package 'Custom Scenery/z_ao_eur/'.
2:58:26.548 E/SCN: Custom Scenery/z_ao_eur/Earth nav data/+40+010/+46+014.dsf:
2:58:26.548 E/SCN: The terrain file 'terrain18.ter' could not be located.
2:58:26.548 E/SCN: DSF canceled due to missing art assets, file=Custom Scenery/z_ao_eur/Earth nav data/+40+010/+46+014.dsf, err=15 (dsf_ErrCanceled).
2:58:26.548 E/SCN: WARNING: got error on DSF load for scenery file 46, 14, Custom Scenery/z_ao_eur/
2:58:26.548 E/SYS: MACIBM_alert: The following scenery package has a problem and will therefore not be loaded:
2:58:26.548 E/SYS: MACIBM_alert: Custom Scenery/z_ao_eur/
2:58:26.548 E/SYS: MACIBM_alert: It requires an additional library scenery package that is not installed, or it is missing some of its files.
2:58:26.548 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_11_/missing_custom_scenery.html
2:58:26.548 E/SYS: MACIBM_alert: /jenkins/design-triggered/source_code/app/X-Plane-f/../../engine/REN/io_dsf.cpp:705
2:58:26.548 E/SYS: +-------------------------------------------------------------------------------
2:58:26.548 E/SYS: | The following scenery package has a problem and will therefore not be loaded:
2:58:26.548 E/SYS: | Custom Scenery/z_ao_eur/
2:58:26.548 E/SYS: | It requires an additional library scenery package that is not installed, or it is missing some of its files.
2:58:26.548 E/SYS: | http://lookup.x-plane.com/_lookup_11_/missing_custom_scenery.html
2:58:26.548 E/SYS: | (io_dsf.cpp:705)
2:58:26.548 E/SYS: +-------------------------------------------------------------------------------
